DirectProgramming/C++SYCL_FPGA/Tutorials/Features/hls_flow_interfaces/mmhost/part4_ddr_hosts/src/mmhost.cpp

```diff
@@ -72,9 +76,14 @@ int main(void) {
     int *array_b = sycl::aligned_alloc_shared<int>(
```

Comment:
```suggestion
    int *array_b = sycl::aligned_alloc_shared<int>(
```

Why aren't we using `alloc_annotated` for this as well? I think it's confusing to mix code styles like this.

Reply:
Updated